packages/editor/src/components/properties/SystemNodeEditor.tsx

```diff
@@ -130,9 +130,10 @@ export const SystemNodeEditor: EditorComponentType = (props) => {
       </InputGroup>
       <InputGroup name="args" label={t('editor:properties.systemnode.lbl-args')}>
         <StringInput
-          onChange={updateProperty(SystemComponent, 'args') as any}
           value={systemComponent.args as any}
-          onRelease={commitProperty(SystemComponent, 'args') as any}
+          onRelease={(value) => {
```

Comment:
this must be commitProperty for it to propagate into snapshots and history

Reply:
this was casuing check-errors to fail so i made this change
can we merge this in and find a fix for this, or should i revert the commit which changes the systemNodeEditor 
we will have to over look the failing npm run check-erorr though

Reply:
moreover since the args key was commented out on the system component , we cannot use commit property on it any longer any ways, i didnt want to remove the code, so i used setComponent instead

Reply:
this is resolved in a separate commit already. please revert these changes and merge from dev